tests/queries/0_stateless/02892_rocksdb_trivial_count.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE dict (key UInt64, value String) ENGINE = EmbeddedRocksDB PRIMARY KEY key;
```

Comment:
Build in fast tests doesn't support rocksdb
```suggestion
-- Tags: use-rocksdb

CREATE TABLE dict (key UInt64, value String) ENGINE = EmbeddedRocksDB PRIMARY KEY key;
```